src/tests/system/tests/test_ldap.py

```diff
@@ -336,20 +299,21 @@ def test_ldap__authenticate_user_with_empty_ldap_search_base(client: Client, lda
         f"Setting option [ldap_group_search_base] to [{base}]",
         f"Setting option [ldap_netgroup_search_base] to [{base}]",
     ]:
-        assert doc in str(log)
+        assert doc in str(log), "Message not found in logs!"
```

Comment:
Should we mention which message was not found in the logs?

This applies also to a few more tests below.

Reply:
Maybe the string should be enclosed in quotation marks? Otherwise we will difficult to identify the limits of each one. We'll have something like:

> String Setting option [ldap_search_base] to [ou=users] not found in logs!